src/main/java/com/poli/internship/api/context/JWTService.java

```diff
@@ -4,24 +4,24 @@
 import com.auth0.jwt.JWTVerifier;
 import com.auth0.jwt.algorithms.Algorithm;
 import com.auth0.jwt.interfaces.DecodedJWT;
-import com.poli.internship.domain.models.AuthTokenPayload;
+import com.poli.internship.domain.models.AuthTokenPayloadModel;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 
 import java.time.Instant;
-import java.util.Map;
 
 @Service
 public class JWTService {
     @Value("${auth.crypto-secret}")
     private String authCryptoSecret;
 
-    public String createAuthorizationToken(AuthTokenPayload tokenPayload) {
+    public String createAuthorizationToken(AuthTokenPayloadModel.AuthTokenPayload tokenPayload) {
```

Comment:
Mudando o import pra `static` vc consegue usar assim:
 ```suggestion
    public String createAuthorizationToken(AuthTokenPayload tokenPayload) {
```